src/app/(views)/catalogs/[catalogId]/edit/add-channel-playlist.tsx

```
@@ -0,0 +1,216 @@
+                {videoLink.error}
+              </p>
+            ) : null}
+          </div>
+
+          {channelInfo.title && channelInfo.id ? (
+            <section className="flex flex-col gap-3">
+              <h1 className="text-base">
+                <a
+                  href={`https://www.youtube.com/channel/${channelInfo.id}`}
+                  className="text-primary hover:underline hover:underline-offset-2"
+                  target="_blank"
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)